tests/installation/installation_overview.pm

```diff
@@ -32,6 +32,21 @@ sub run {
         assert_screen "inst-xen-pattern";
     }
 
+    # preserve it for the video
+    wait_idle 10;
```

Comment:
delete. We don't allow "wait_idle" anymore. A rebase should provide a check that fails whenever someone uses a wait_idle